package.json

```diff
@@ -8,7 +8,7 @@
     "build": "ng build",
     "build:stage": "ng build -c staging",
     "build:prod": "ng build -c production",
-    "test": "ng test",
+    "test": "ng test --browsers=ChromeHeadless",
```

Comment:
Did you mean to add this? I don't mind it, but I think this should be another command, so we can easily run test with chrome as well.